### PR TITLE
refs #114: additions mentioned in my comments

### PR DESCRIPTION
**Pull request by stickgrinder**
@paolomainardi here is my personal additions plus a bit of rephrasing here and there, as mentioned in my comments on documentation and project-level checkmarks.

I guess a run on grammarly may help, I didn't to it yet.
Let me know if you need me to change stuff.

### CODE REVIEW

File: content/procedures/universal-dod.md [hunk 1]
```diff
@@ -4,48 +4,68 @@ _/
 
 We love to build **high-quality products**, always following the market **best practices** regarding **tools**, **security**, **coding**, and **team collaboration**.
 
-To better **match our standards** in all the **technologies** we work with, we have in place **several practices** to help **everyone** to be **successful**:
+To better **match our standards** in all the **technologies** we work with, we have in place **several practices** to help **everyone** being **successful**:
```
Comment (aijcoa) [inline content/procedures/universal-dod.md:7]: This was sentence was fine, `to be` makes sense.

File: content/procedures/universal-dod.md [hunk 2]
```diff
@@ -4,48 +4,68 @@ _/
...
-2.  Automation from the local development environment to the CI/CD pipelines (Gitlab and GitHub).
+1.  Starter-kits (our custom package system) to quickly bootstrap new projects
+2.  Automation from the local development environment to the CI/CD pipelines (Gitlab and GitHub)
 3.  Code review
 4.  Upskilling and training program
 5.  Internal tech talks and company hackathon
 6.  On-demand QA team to help testing features
 
-Any of the following items allows you to shape a **better product**, reducing the **cognitive load** needed to **build, test, and deliver** **digital products** compatible with the **company standard**.
+All this helps us shaping a **better product**, reducing the **cognitive load** needed to **build, test, and deliver** **digital products** compatible with the **company standards**.
```
Comment (aijcoa) [inline content/procedures/universal-dod.md:16]: All this helps us shaping -> All this helps us **to shape**

File: content/procedures/universal-dod.md [hunk 3]
```diff
@@ -4,48 +4,68 @@ _/
...
+### What's a Definition of Done?
 
 According to the _Scrum Alliance_ definition:
 
 > _"Definition of done is a_ **_simple list of activities_** _(writing code, coding comments, unit testing, integration testing, release notes, design documents, etc.)_ **_that add verifiable/demonstrable value_** _to the_ **_product_**_._ **_Focusing_** _on_ **_value-added steps_** _allows the team to focus on what_ **_must be completed_** _in order to build software while_ **_eliminating wasteful activities_** _that only complicate software growth efforts."_
 
 _Source:_ [_https://resources.scrumalliance.org/Article/definition-dod_](https://resources.scrumalliance.org/Article/definition-dod)
 
-Well, the concept behind this is quite simple: having **a checklist of valuable tasks** to ensure to be on the **right path** and understand if a **feature is effectively done**.
+In simple words a _Definition of Done_ is **a checklist of non-functional requirements** that ensure to be on the **right path** and understand if a **task is actually done** or if it needs more work.
```
Comment (aijcoa) [inline content/procedures/universal-dod.md:26]: that ensure -> **to ensure**

File: content/procedures/universal-dod.md [hunk 4]
```diff
@@ -4,48 +4,68 @@ _/
...
-- **Acceptance criteria, when present on the issue, are all met.**
+- The project can be entirely built locally by running `make` on a correctly and freshly provisioned computer.
+- All the operations to deploy the project to non-local environments are automated and don't depend on manual task or personal knowledge.
+- All information to access the environments are properly documented (see above).
+- No mocked or half-baked features are deployed to the staging or production environments.
 
 ### Conclusion
 
-This is a general list of **rules that can apply to all kinds of scenarios**; it **doesn't cover specific use cases** that must be part of the **project itself**, **agreed** upon by the **team members**, and **constantly reviewed** to keep it **valuable and doable**.
+This is a general list of **non-functional rules that apply to every scenarios**; with _non-functional_ we mean **they are not bound to specific use cases** but to the **project itself**, **agreed** upon by the **team members**, and **constantly reviewed** to keep it **valuable and doable**.
```
Comment (aijcoa) [inline content/procedures/universal-dod.md:66]: every scenarios > every scenario

**Comment by aijcoa** [comment]
Added a couple of suggestions for the grammar, but it all looks good. I can take a look at the rest of the playbook this week if you'd like 👍

**Review by paolomainardi — CHANGES_REQUESTED** [review]
Thanks a lot @aijcoa 

@stickgrinder, thanks for the addition; once the wording fixes are in place, we can merge this.

**Comment by stickgrinder** [comment]
@paolomainardi I fixed all mistakes (thanks @aijcoa) and moved the file to the "Policies" section (since DoD is actually a policy).
Thanks

**Review by paolomainardi — APPROVED** [review]
LGTM

**Comment by paolomainardi** [comment]
Done! Thanks @stickgrinder and @aijcoa

**Pull request merged**